### PR TITLE
en,zh: add v1.5.1 release notes

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

- add v1.5.1 release notes

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.5.1.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- Support replacing volumes for PD, TiKV and TiDB ([#5150](https://github.com/pingcap/tidb-operator/pull/5150), [@anish-db](https://github.com/anish-db))
+
+## Improvements
+
+## Bug fixes
+
+- Fix errors from PVC modifier when a manual TiKV eviction is requested ([#5302](https://github.com/pingcap/tidb-operator/pull/5302), [@anish-db](https://github.com/anish-db))
+- Fix a deadlock in TiDB Operator casued by the TiKV eviction when a volume replacement is ongoing ([#5301](https://github.com/pingcap/tidb-operator/pull/5301), [@anish-db](https://github.com/anish-db))
+- Fix the issue that TidbCluster can't be rolled back when upgrading ([#5345](https://github.com/pingcap/tidb-operator/pull/5345), [@anish-db](https://github.com/anish-db))
```
Comment (ran-huang) [inline en/releases/release-1.5.1.md:21]:
```suggestion
- Fix the issue that TidbCluster may be able to roll back during the upgrade process ([#5345](https://github.com/pingcap/tidb-operator/pull/5345), [@anish-db](https://github.com/anish-db))
```

Comment (BornChanger) [inline en/releases/release-1.5.1.md:21]:
```suggestion
- Fix the issue that TidbCluster can't be rolled back when upgrading ([#5345](https://github.com/pingcap/tidb-operator/pull/5345), [@anish-db](https://github.com/anish-db))
- Fix the problem of MaxReservedTime doesn't work for scheduled backup ([#5147](https://github.com/pingcap/tidb-operator/pull/5147), [@BornChanger](https://github.com/BornChanger))
```

File: zh/releases/release-1.5.1.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- 支持替换 PD、TiKV 以及 TiDB 所使用的 volume ([#5150](https://github.com/pingcap/tidb-operator/pull/5150), [@anish-db](https://github.com/anish-db))
+
+## 优化提升
+
+## Bug 修复
+
+- 修复手动触发 TiKV eviction 时 PVC Modifier 报错的问题 ([#5302](https://github.com/pingcap/tidb-operator/pull/5302), [@anish-db](https://github.com/anish-db))
+- 修复替换 TiKV volume 过程中再触发 TiKV eviction 时可能造成 TiDB Operator reconcile 死锁的问题 ([#5301](https://github.com/pingcap/tidb-operator/pull/5301), [@anish-db](https://github.com/anish-db))
+- 修复 TidbCluster 在 Upgrade 过程中可能无法回滚的问题 ([#5345](https://github.com/pingcap/tidb-operator/pull/5345), [@anish-db](https://github.com/anish-db))
```
Comment (BornChanger) [inline zh/releases/release-1.5.1.md:21]:
```suggestion
- 修复 TidbCluster 在 Upgrade 过程中可能无法回滚的问题 ([#5345](https://github.com/pingcap/tidb-operator/pull/5345), [@anish-db](https://github.com/anish-db))
- 修复 MaxReservedTime 选项没有被 backup schedule gc 使用的问题 [#5147](https://github.com/pingcap/tidb-operator/pull/5147), [@BornChanger](https://github.com/BornChanger))
```

File: en/releases/release-1.5.1.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Support replacing volumes for PD, TiKV, and TiDB ([#5150](https://github.com/pingcap/tidb-operator/pull/5150), [@anish-db](https://github.com/anish-db))
+
+## Improvements
+
+## Bug fixes
+
+- Fix errors from PVC modifier when a manual TiKV eviction is requested ([#5302](https://github.com/pingcap/tidb-operator/pull/5302), [@anish-db](https://github.com/anish-db))
+- Fix a deadlock issue in TiDB Operator caused by the TiKV eviction when a volume replacement is ongoing ([#5301](https://github.com/pingcap/tidb-operator/pull/5301), [@anish-db](https://github.com/anish-db))
+- Fix the issue that TidbCluster may be able to roll back during the upgrade process ([#5345](https://github.com/pingcap/tidb-operator/pull/5345), [@anish-db](https://github.com/anish-db))
+- Fix the problem of MaxReservedTime doesn't work for scheduled backup ([#5148](https://github.com/pingcap/tidb-operator/pull/5148), [@BornChanger](https://github.com/BornChanger))
```
Comment (ran-huang) [inline en/releases/release-1.5.1.md:22]:
```suggestion
- Fix the issue that the `MaxReservedTime` option does not work for scheduled backup ([#5148](https://github.com/pingcap/tidb-operator/pull/5148), [@BornChanger](https://github.com/BornChanger))
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Pull request merged**